recipe/meta.yaml

```diff
@@ -1,44 +1,64 @@
-{% set version = "2018_20171205" %}
+{% set vmajor = 2018 %}       # load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_VERSION_MAJOR +(\d+)").group(1)|int
+{% set vminor = 0 %}          # load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_VERSION_MINOR +(\d+)").group(1)|int
+{% set vinterface =  10003 %} # load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_INTERFACE_VERSION +(\d+)").group(1)|int
+{% set version = "%d.%d.%d"%(vmajor,vminor,vinterface%1000) %}
+{% set args=' --build-args=stdver=c++11 --install-libs --install-devel --install-docs' %} #  no --install-python yet
 
 package:
   name: tbb
   version: {{ version }}
 
 source:
-  fn: tbb{{ version }}oss_src.tgz
-  url: https://github.com/01org/tbb/archive/2018_U2.tar.gz
-  sha256: 78bb9bae474736d213342f01fe1a6d00c6939d5c75b367e2e43e7bf29a6d8eca
+  git_url: https://github.com/01org/tbb.git
+  git_tag: {{ vmajor }}_U{{ vinterface%1000 }}
 
 build:
-  number: 0
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  script: python build/build.py {{ args }}                                          # [unix]
+  script: python build/build.py --make-tool=mingw32-make {{ args }}                 # [win]
+  #script: env MACOSX_DEPLOYMENT_TARGET=10.7 python build/build.py {{ args }}       # [osx]
+  script_env:
+    - OS    # [win]
```

Comment:
What's this needed for?

Reply:
TBB's makefiles detect platform using this variable commonly set on Windows. Apparently, conda-build removes this variable unless specified otherwise (conda's bug?). Though, in the recent version of build.py we work it around as well, so, not so important to have it here anymore.. except if somebody wants to build TBB directly using Makefiles and without using build.py